client/src/app/slides/motions/motion/motions-motion-slide.component.html

```diff
@@ -1,4 +1,38 @@
 <div *ngIf="data">
-    Motion Slide
-    <h1>{{ data.data.title }}</h1>
+    <div id="sidebox" *ngIf="data.data.show_meta_box">
+        <!-- State -->
+        <h3 translate>State</h3>
+        {{ data.data.state }}
+        <span *ngIf="data.data.state_extension">{{ data.data.state_extension }}</span>
+
+        <!-- Recommendation -->
+        <div *ngIf="data.data.recommendation">
+            <h3 translate>Recommendation</h3> <!-- TODO: use config 'motions_recommendations_by'-->
+            {{ data.data.recommendation }}
+        </div>
+
+        <!-- Submitters -->
+        <h3 translate>Submitters</h3>
+        <div *ngFor="let submitter of data.data.submitter">
+            <!-- TODO: order by weight-->
+            {{ submitter }}
```

Comment:
@FinnStutzenstein Any idea how I can get the submitter name?

Reply:
The server gives us a "small user representation". Use all given fields to make full/short name. If fields are missing, add them in the server.

The user model does have implementations for full/short name. Move them out of the model, so you can use them from the model and the slides. You must just add a few parameters for all needed things (username, firstname, title, ...)

Reply:
Will be fixed in #4194